test/parallel/test-vm-sigint.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const vm = require('vm');
+
+const spawn = require('child_process').spawn;
+
+if (process.argv[2] === 'child') {
+  const parent = +process.env.REPL_TEST_PPID;
+  assert.ok(parent);
+
+  try {
+    vm.runInThisContext(`process.kill(${parent}, "SIGUSR2"); while(true) {}`, {
+      breakOnSigint: true
+    });
+  } catch (e) {
+    assert.ok(e);
+    process.stdout.write(e.message);
+    process.exit(0);
+  }
+
+  process.exit(1);
```

Comment:
Can you replace this with e.g. `common.fail('Expected exception missing')`?  Something that prints an error in case of failure.